crates/uv/tests/pip_compile.rs

```diff
@@ -7768,11 +7768,11 @@ fn universal_transitive_disjoint_prerelease_requirement() -> Result<()> {
     ----- stdout -----
     # This file was autogenerated by uv via the following command:
     #    uv pip compile --cache-dir [CACHE_DIR] requirements.in --universal
-    cffi==1.16.0 ; platform_python_implementation != 'PyPy' or os_name == 'linux'
+    cffi==1.16.0 ; os_name == 'linux'
         # via
         #   -r requirements.in
         #   cryptography
-    cffi==1.17.0rc1 ; os_name != 'linux' or platform_python_implementation != 'PyPy'
+    cffi==1.17.0rc1 ; os_name != 'linux'
```

Comment:
I think this was wrong before...? The requirements are:

```
cffi ; os_name == 'linux'
cffi >= 1.17.0rc1 ; os_name != 'linux'
cryptography
```

Reply:
Or is it just that these are redundant...? Since `os_name == 'linux' or os_name != 'linux'` already covers the full space?

Reply:
It is definitely redundant in terms of the marker expressions. But it's not immediately obvious to me why this change dropped that part of the marker.